news-aggregator-api/src/controllers/getNews.js

```diff
@@ -0,0 +1,40 @@
+const fetchUrl = require("../helpers/fetchUrl");
+const { newsFromJSON,categories } = require("../models/newsModel");
+const { writeToFile } = require("../helpers/fileOperations");
+const URL = "https://newsapi.org/v2/";
+
+
+/**
+ * To fetch the news globally, here used for cron-job to keep news refreshing every 14 minutes,
+ * which is ~(24*60)/100
+ * 100 is number of requests available for free by newsapi.org
+ *
+ * it just writes to file
+//  */
+let categoriesList = categories()
+
+async function getNews(){
+  for await (const category of categoriesList) {
+    let payload = {
+      page: 1,
+      pageSize : 50,
+      q: category,
+      apiKey: process.env.NEWS_API_KEY,
```

Comment:
for this, try to maintain a .env file and list out all the required variables
commit only a sample copy and keep sensitive data locally

Reply:
Sure Anuj, Thanks